### PR TITLE
v0.3

### PR DESCRIPTION
**Pull request by julien-f**

### CODE REVIEW

File: src/index.spec.js [hunk 1]
```diff
@@ -28,6 +28,14 @@ const data = deepFreeze({
       username: 'Administrator',
       password: 'pas:sw@ord'
     }
+  },
+  NFS: {
+    string: 'nfs://192.168.100.225:media/nfs',
```
Comment (julien-f) [inline src/index.spec.js]: Is there no leading slash for the path?

What is currently done?

Comment (marsaud) [inline src/index.spec.js]: Ah yes there is practice, but it is not under responsibility of the parser, the test data could just be SOME_HOST:SOME_PATH in fact.

Wait, I'm doing some checks...

**Comment by julien-f** [comment]
Feel free to assign me to this PR when it's good enough for you and you want me to make a release.

File: src/index.spec.js (hunk 1 above)
Comment (marsaud) [inline src/index.spec.js]: Ugly. The path is stored without a leading slash, and the / is added just in time by xo-server before mounting.

Comment (julien-f) [inline src/index.spec.js]: That's probably something that should be fixed both in the parser and the server…

Comment (marsaud) [inline src/index.spec.js]: Should the parser be able to "fix" this for previous created remotes, and I remove the extra slash on server side with remote parser 3 a s a dependency ?

Comment (julien-f) [inline src/index.spec.js]: Seems like the good way to fix it.

**Comment by julien-f** [comment]
xrp v0.3.0 published!

**Pull request closed without merging**